readme.md

```diff
@@ -81,6 +86,10 @@ As part of package creation, this plugin will create three shell scripts:
    - `stop`: if the process status is 0, issues a kill signal to the process.
  * `service/bin/config.sh`: a shell script containing environment variables to apply
     as overrides when `init.sh` is run.
+ * `service/monitoring/bin/check.sh`: a no-argument shell script that returns `0` when
```

Comment:
Actually, we use 'returns' in the description on 'status'. I'll leave this as is for now.